contracts/DefaultStrategy.sol

```diff
@@ -0,0 +1,497 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.11;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
+
+import "./common/UsingRegistryUpgradeable.sol";
+import "./common/UUPSOwnableUpgradeable.sol";
+import "./interfaces/IAccount.sol";
+import "./interfaces/IGroupHealth.sol";
+import "./Managed.sol";
+import "./interfaces/IManager.sol";
+import "./interfaces/IAllowedStrategy.sol";
+
+contract DefaultStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    /**
+     * @notice Holds a group's address and votes.
+     * @param group The address of the group.
+     * @param votes The votes assigned to the group.
+     */
+    struct GroupWithVotes {
+        address group;
+        uint256 votes;
+    }
+
+    /**
+     * @notice Contract used for checking group health and rebalancing.
+     */
+    IGroupHealth public groupHealthContract;
+
+    /**
+     * @notice An instance of the Account contract this Manager manages.
+     */
+    IAccount internal account;
+
+    /**
+     * @notice Contract used for checking group health and rebalancing.
+     */
+    IAllowedStrategy public allowedStrategyContract;
+
+    /**
+     * @notice Used when attempting to withdraw from allowed strategy
```

Comment:
Consider "Used" -> "Emitted" here and throughout for consistency with the code base.

Reply:
I changed all events to "Emitted" but left all error as "Used" since it is like this also in Account, Manager and so on

Reply:
Ah right, missed that this is an error, not an event. My bad.